client/src/app/+my-account/my-account-notifications/my-account-notifications.component.ts

```diff
@@ -8,6 +8,8 @@ import { UserNotificationsComponent } from '@app/shared/shared-main'
 export class MyAccountNotificationsComponent {
   @ViewChild('userNotification', { static: true }) userNotification: UserNotificationsComponent
 
+  notificationSortType = 'created'
```

Comment:
Where do you use this variable (except for the select)?

Reply:
I changed it.